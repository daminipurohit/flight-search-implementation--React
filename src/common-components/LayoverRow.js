import React from 'react';
import PropTypes from 'prop-types';
import '../styles/layoverRow.scss';

const LayoverRow = ({ layoverTime }) => (
    <div className="layover-wrapper">
        <span className="layover-text">{layoverTime}</span>
        <div className="horizontal-dash" />
    </div>
);

LayoverRow.propTypes = {
    layoverTime: PropTypes.string.isRequired
};

export default LayoverRow;
