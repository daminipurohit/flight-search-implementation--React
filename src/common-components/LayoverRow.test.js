import React from 'react';
import { shallow } from 'enzyme';
import LayoverRow from './LayoverRow';

describe('LayoverRow', () => {
    it('should render correctly', () => {
        const component = shallow(<LayoverRow layoverTime="2hr" />);
        expect(component).toMatchSnapshot();
    });
});
