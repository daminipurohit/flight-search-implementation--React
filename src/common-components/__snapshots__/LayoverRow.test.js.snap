// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoverRow should render correctly 1`] = `
<div
  className="layover-wrapper"
>
  <span
    className="layover-text"
  >
    2hr
  </span>
  <div
    className="horizontal-dash"
  />
</div>
`;
